lighthouse-core/audits/seo/manual/mobile-friendly.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const ManualAudit = require('../../manual/manual-audit');
+
+/**
+ * @fileoverview Manual SEO audit to check if page is mobile friendly.
+ */
+
+class MobileFriendly extends ManualAudit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return Object.assign({
+      name: 'mobile-friendly',
+      helpText: 'Take the [Mobile-Friendly Test](https://search.google.com/test/mobile-friendly) to see how easily a visitor can use your page on a mobile device. [Learn more](https://developers.google.com/search/mobile-sites/).',
```

Comment:
given that LH is all about mobile web advice, I'm assuming the plan is to convert this into an automatic test at some point? :)

Reply:
As discussed, this is currently using these 6 rules in the check: https://support.google.com/webmasters/answer/6352293?utm_source=search_console&utm_campaign=sc-mft-test

once we have coverage for all of them in LH, we can migrate this to a multicheckaudit and drop the recommendation to use MFT as we'll be testing the exact same thing. (we could also validate there is no significant difference in results across x000 URLs)